backend/apis/clients/list-clients.api.js

```diff
@@ -146,7 +145,7 @@ app.get("/api/clients", (req, res, next) => {
         firstName: row.firstName,
         lastName: row.lastName,
         fullName: responseFullName(row.firstName, row.lastName),
-        birthday: row.birthday,
+        birthday: responseDateWithoutTime(row.birthday),
```

Comment:
We should only send down "1990-01-01" for birthdays. Not "1990-01-01T700:00:00.00Z"